src/sql/workbench/browser/modal/modal.ts

```diff
@@ -173,7 +173,7 @@ export abstract class Modal extends Disposable implements IThemable {
 		this._bodyContainer = DOM.$(`.${builderClass}`, { role: 'dialog', 'aria-label': this._title });
 		const top = this.layoutService.getTitleBarOffset();
 		this._bodyContainer.style.top = `${top}px`;
-		this._modalDialog = DOM.append(this._bodyContainer, DOM.$('.modal-dialog', { role: 'document' }));
```

Comment:
did you look into the history of this? why was this added?

Reply:
It's been there since the beginning of the file history in 2017 in the Public Preview 1 commit 3cdac4133909dc32df9223fa4f038b340d266b9d. Is there somewhere I can find the history from before that?

Reply:
@alanrenmsft one more question - any hunch what could be the impact (unwanted) of removing this?

Reply:
Does this help at all? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Document_Role

Reply:
@kevcunnane  Do you know how to view the old sqlops repo, we have added redirection, i can't get to it directly :( 

@udeeshagautam I assume it is something related to accessibility, but not sure, to be safe, lets dig into the sqlops code base, if we couldn't find the old history, I guess this is fine since they are re-testing the dialog.

Reply:
I'll send it on. The "sqlops" got renamed, but we do still have private pre-history I can share 😄

Reply:
This has been there since the file modal.ts was added in inhttps://github.com/microsoft/carbon/pull/1331

Reply:
I got a little bit further by following the file that was deleted in the PR you send, but lost track again. I am fine with this change.